packages/server/events-processing-platform/subscriptions/organization/domain_scraper.go

```diff
@@ -84,7 +94,7 @@ func (ds *DomainScraperV1) getHtml(domainUrl string, directGet bool) (*string, e
 			return nil, errors.Wrap(err, "failed to execute request")
 		}
 	} else {
-		response, err = ds.proxyGetRequest(ds.cfg.Services.ScrapingBeeApiKey, domainUrl)
+		response, err = ds.proxyGetRequest(ds.cfg.Services.ScrapingBeeApiKey, domainUrl, httpClient)
```

Comment:
shouldn't the key be passed as arg? shouldn't we called linkedin with SCRAPING_DOG_API_KEY?

Reply:
Edi, good catch, I added that and forgot to come back and change after I wrote more code 😵